src/applications/personalization/profile360/sass/profile-360.scss

```diff
@@ -82,9 +82,6 @@ input.usa-only-phone {
   width: 100%;
   padding: 0 5px;
 
-  .address-validation-input {
```

Comment:
No idea why this was done, but I'm always happy to remove a hard-coded dimension like this!

Reply:
oh no... [looks like this was done for IE11](https://github.com/department-of-veterans-affairs/vets-website/commit/86771289b1810451e6008413c178a7f8fbd63a46) :( So this might reintroduce a bug in IE11 if it's removed.

Reply:
Interesting - I did test just now on my Windows on IE11 and I don't see any issues removing the height. The entire section of the address is still clickable and successfully selects the desired address.